.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,17 +14,25 @@
 
 Closes # .
 
+<!-- The next section is mandatory. If your PR doesn't require testing, please indicate that you are purposefully omitting instructions.  Otherwise, please include detailed instructions on how these changes can be tested (including pre-conditions, configuration, steps to take and expected results). It may help to write your instructions using pseudocode -- as if you're telling a computer how to execute the test.-->
```

Comment:
To match the location of the other comments in the template, maybe this should be `This section is mandatory` and the comments and new task moved to the `### How to test the changes in this Pull Request:` section

Reply:
Good call.